src/components/ChangePlan/Checkout/Checkout.test.js

```diff
@@ -6,6 +6,7 @@ import IntlProvider from '../../../i18n/DopplerIntlProvider.double-with-ids-as-v
 import { AppServicesProvider } from '../../../services/pure-di';
 import { BrowserRouter } from 'react-router-dom';
 import Checkout from './Checkout';
+import { fakeIndustries } from '../../../services/static-data-client.double';
 
 describe('Checkout component', () => {
   afterEach(cleanup);
```

Comment:
jest automatically does the cleanup, so you don't need to do it manually

Reply:
Done